packages/itwin/grouping-mapping-widget/src/components/Mappings/Extraction/ExtractionStatusIcon.tsx

```diff
@@ -6,8 +6,13 @@ import { IconButton, NotificationMarker, Tooltip } from "@itwin/itwinui-react";
 import React from "react";
 import { SvgProcess } from "@itwin/itwinui-icons-react";
 
-export interface ExtractionStatusIconProps {
+export interface ExtractionStatusData {
   iconStatus: "negative" | "positive" | "warning" | undefined;
+  iconMessage: string;
+}
+
+export interface ExtractionStatusIconProps {
+  iconStatus: ExtractionStatusData["iconStatus"];
```

Comment:
Maybe it would be better to create a separate type for the `iconStatus` instead of interface properties referencing one another.

Reply:
I agree. Done.